package.json

```diff
@@ -17,6 +17,7 @@
     "create-react-class": "^15.7.0",
     "csstype": "^3.0.11",
     "customize-cra": "^1.0.0",
+    "packages": "^0.0.8",
```

Comment:
It made a re-appearance! Could you remove it again?

Reply:
I figured out what I was doing - whoops. Won't appear again.